lib/ui/widgets/components/scrollbar.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+
+class ScrollBar extends StatefulWidget {
```

Comment:
That name could be confused with `Scrollbar`. 

Maybe we should name it `ArchethicScrollbar` or `MultiplatformScrollbar` , `ImprovedScrollbar` ?